src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  box-sizing: border-box;
+  height: 408px;
```

Comment:
Don't hardcode height. Don't fix container size (if there is no such a requirement)
<img width="561" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/fd78ea12-f7b0-4a1a-9053-c175164efc32">

Reply:
Removed